README.md

```diff
@@ -18,14 +18,20 @@ This will install the runner as `codemod`. This package requires node v6 or high
 
 The primary interface is as a command line tool, usually run like so:
 
-```
+```sh
 $ codemod --plugin transform-module-name \
   path/to/file.js \
   another/file.js
 ```
 
 This will re-write the files `path/to/file.js` and `another/file.js` by transforming them with the babel plugin `transform-module-name`. Multiple plugins may be specified, and multiple file or directories may be re-written at once.
 
+Plugins may also be loaded from [AST Explorer](https://astexplorer.net/) using `--remote-plugin`. This feature should only be used as a convenience to load code that you or someone you trust wrote. It will run with your full user privileges, so please exercise caution!
```

Comment:
The way I wrote it you can actually load things from any URL that contains a babel plugin, but there’s special handling for astexplorer.net. I should probably document that here.